src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/Generated/Models/DeploymentProperties.cs

```diff
@@ -29,7 +29,7 @@ public partial class DeploymentProperties
         /// <summary>
         /// Initializes a new instance of the DeploymentProperties class.
         /// </summary>
-        public DeploymentProperties(object template = default(object), TemplateLink templateLink = default(TemplateLink), object parameters = default(object), ParametersLink parametersLink = default(ParametersLink), DeploymentMode? mode = default(DeploymentMode?), DebugSetting debugSetting = default(DebugSetting))
+        public DeploymentProperties(DeploymentMode? mode, object template = default(object), TemplateLink templateLink = default(TemplateLink), object parameters = default(object), ParametersLink parametersLink = default(ParametersLink), DebugSetting debugSetting = default(DebugSetting))
         {
```

Comment:
@ravbhatnagar @TianoMS Spoke to @amarzavery There is no way to set default value for enum for a model property (in our case, mode) in c# yet. Although, it looks like other languages can support it. I'll add the default to our spec in a separate PR, so this can move forward. @hovsepm changes look good to you?